securedrop/Makefile

```diff
@@ -0,0 +1,37 @@
+TAG ?= $(shell git rev-parse HEAD)
+IMAGE = securedrop
+
+.PHONY: assets
+assets:
+	sass
+
+.PHONY: images
+images:
+	docker build $(EXTRA_BUILD_ARGS) -t $(IMAGE):$(TAG) .
+	# combine dockerfiles but strip out first COPY for better cache utilization
+	cat Dockerfile* |sed '/#skip-if-test$$/N;/#skip-if-test/d' | docker build $(EXTRA_BUILD_ARGS) -f - -t $(IMAGE)-test:$(TAG) .
+
+.PHONY: test
+test:
+	docker run --privileged \
```

Comment:
why not **--rm** here ? Are there situations where we want to keep the container after a run ?

Reply:
it should use --rm.

Reply:
actually the caching of layers in circle relies on this.